apollo-normalized-cache/src/commonMain/kotlin/com/apollographql/apollo/cache/normalized/simple/SimpleNormalizedCache.kt

```diff
@@ -0,0 +1,45 @@
+package com.apollographql.apollo.cache.normalized.simple
+
+import com.apollographql.apollo.cache.CacheHeaders
+import com.apollographql.apollo.cache.normalized.CacheKey
+import com.apollographql.apollo.cache.normalized.NormalizedCache
+import com.apollographql.apollo.cache.normalized.Record
+
+class SimpleNormalizedCache : NormalizedCache() {
```

Comment:
internal? Also should we call it `InMemoryNormalizedCache` or  `MapNormalizedCache`?

Reply:
Renamed to `MapNormalizedCache`. `InMemoryNormalizedCache` might clash with the future LRU cache.